tsdb/head.go

```diff
@@ -381,6 +422,20 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_snapshot_replay_error_total",
 			Help: "Total number snapshot replays that failed.",
 		}),
+		oooHistogram: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Name: "prometheus_tsdb_sample_ooo_delta",
+			Help: "Delta in seconds by which a sample is considered out of order (reported regardless of OOO time window and whether sample is accepted or not).",
+			Buckets: []float64{
+				// Note that mimir distributor only gives us a range of wallclock-12h to wallclock+15min
```

Comment:
Not sure if this comment should be here - perhaps something for Mimir docs. Happy with Mimir friendly buckets for now, unless Prometheus or other projects will need different ones in future.

Reply:
Ahhh, yeah the comment needs to be updated. Once we have the native histograms, we should not have this problem of choosing buckets :)

Reply:
I think the buckets makes sense without the Mimir context as well.